presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestUnweightedDoubleReservoirSample.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.reservoirsample;
+
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+
+public class TestUnweightedDoubleReservoirSample
+{
+    @Test
+    public void testGetters()
```

Comment:
- Rename to `testGetMaxSamples` for clarity
- Drop `final`
- Put the entirety of first statement on a single line

```
    @Test
    public void testGetMaxSamples()
    {
        UnweightedDoubleReservoirSample sample = new UnweightedDoubleReservoirSample(200);
        assertEquals(sample.getMaxSamples(), 200);
    }
```

Reply:
Removed `final` everywhere.